docs/ReleaseNotes.md

```diff
@@ -1,6 +1,32 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 3.1.2
+Acuminator 3.1.2 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 3.1.2, the following enhancements have been implemented:
+ - The [PX1096](diagnostics/PX1096.md) diagnostic has been added to check signature of methods with the `PXOverride` attribute.
+ - The [PX1007](diagnostics/PX1007.md) diagnostic no longer checks the following DAC fields for a code annotation:
+   - `Selected`
+   - `Attributes`
+   - `GroupMask`
+   - `CompanyID`
+   - `CompanyMask`
+   - `DeletedDatabaseRecord`
+ - The [PX1007](diagnostics/PX1007.md) diagnostic now checks that mapped DAC fields of projection DACs have the `inheritoc` XML tag
+ 
```

Comment:
The PX1007 changes description should also include that PX1007 supports  `inheritdoc` tag in general, not just for projection DAC mapped properties. Basically, you can put any form of inheritdoc tag on normal DACs, DAC extensions and their properties but you have to follow particular rule for projection DAC mapped DAC properties.